hack/pre-release.sh

```diff
@@ -303,6 +318,92 @@ dist_git_update() {
   echo
 }
 
+# This function handles updating the versions in gitlab. Expects you to be connected to the VPN
+# takes 2 arguments
+# 1st arg: stream to update. Can be x, y or z to represent the x y or z stream.
+# 2nd arg: current WMCO version
+# 3rd arg: updated WMCO version
+gitlab_update() {
+ if [ "$#" -ne 3 ]; then
+    echo incorrect parameter count for dist_git_update $#
+    return 1
+  fi
+  stream=$1
+  wmco_version=$2
+  updated_version=$3
+  OCP_version=$(get_OCP_version $wmco_version)
+
+  current_dir=$PWD
+
+  # clones the gitlab repo into a temporary directory
+  temp=$(mktemp -d)
+  cd $temp
+  clone_repo "https://gitlab.cee.redhat.com/cpaas-products/openshift-winc.git"
+
+  GITLAB_LOCATION=$temp/openshift-winc
+  cd $GITLAB_LOCATION
+
+  latest_hash=$(git rev-parse HEAD)
+
+  gitlab_initial_branch=$(git branch --show-current)
+  new_branch=$(create_git_branch_list $stream-stream-bump $gitlab_initial_branch)
```

Comment:
Is this branch name unique? What happens if this is not deleted after the MR merges?

Reply:
the [switch_git_branch ](https://github.com/openshift/windows-machine-config-operator/blob/713a399a6d085721458c1d602a287ce4edf8cf4c/hack/common.sh#L85) function in common.sh that I use [here](https://github.com/openshift/windows-machine-config-operator/blob/713a399a6d085721458c1d602a287ce4edf8cf4c/hack/pre-release.sh#L350) deletes the branch if there's a duplicate. This is the same function used in the submodule update script and elsewhere in this file.